plugins/lookup/hashi_vault.py

```diff
@@ -161,79 +161,79 @@
 """
 
 EXAMPLES = """
-- debug:
+- ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello:value token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200') }}"
 
 - name: Return all secrets from a path
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200') }}"
 
 - name: Vault that requires authentication via LDAP
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/hello:value auth_method=ldap mount_point=ldap username=myuser password=mypas') }}"
 
 - name: Vault that requires authentication via username and password
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello:value auth_method=userpass username=myuser password=psw url=http://myvault:8200') }}"
 
 - name: Using an ssl vault
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hola:value token=c975b780-d1be-8016-866b-01d0f9b688a5 validate_certs=False') }}"
 
 - name: using certificate auth
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/hi:value token=xxxx url=https://myvault:8200 validate_certs=True cacert=/cacert/path/ca.pem') }}"
 
 - name: authenticate with a Vault app role
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello:value auth_method=approle role_id=myroleid secret_id=mysecretid') }}"
 
 - name: Return all secrets from a path in a namespace
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello token=c975b780-d1be-8016-866b-01d0f9b688a5 namespace=teama/admins') }}"
 
 # When using KV v2 the PATH should include "data" between the secret engine mount and path (e.g. "secret/data/:path")
 # see: https://www.vaultproject.io/api/secret/kv/kv-v2.html#read-secret-version
 - name: Return latest KV v2 secret from path
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/data/hello token=my_vault_token url=http://myvault_url:8200') }}"
 
 # The following examples work in collection releases after community.general 0.2.0
 
 - name: secret= is not required if secret is first
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/data/hello token=<token> url=http://myvault_url:8200') }}"
 
 - name: options can be specified as parameters rather than put in term string
-  debug:
+  ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/data/hello', token=my_token_var, url='http://myvault_url:8200') }}"
 
 # return_format (or its alias 'as') can control how secrets are returned to you
 - name: return secrets as a dict (default)
-  set_fact:
+  ansible.builtin.set_fact:
```

Comment:
Did you test whether `set_fact` actually works as expected with FQCN? I know that there is some special handling code in core for `set_fact`, and it could be that it hasn't been adjusted to also work with FQCN.

Reply:
good point, returned